### ISSUE TITLE
Convert offload.env file to a config file

### ISSUE THREAD

**Issue by nj1973** [gluent/goe#189]
As we move towards supporting Offload commands in cloud architectures we need to decouple the `offload.env` file location from `OFFLOAD_HOME` and remove the reliance on `bash`.

Tasks to be completed for this issue are:

1. Convert `offload.env` to a simpler configuration file (not bash)
2. Update any scripts that currently source the file to not include that action
3. Update any documentation/examples that currently source the file to not include that action

Once this is complete we should create a new issue to support placing the config file in GCS and have Offload support an option (`--config-file`?) to accept a custom location, outside of `OFFLOAD_HOME`.
